docs/querying/scan-query.md

```diff
@@ -228,3 +211,7 @@ Sample query context JSON object:
   "maxSegmentPartitionsOrderedInMemory": 100
 }
 ```
+
+## Legacy mode
+
+In older versions of Druid, the scan query supported a legacy mode designed for protocol compatibility with the former scan-query contrib extension from versions of Druid older than 0.11. This mode has been removed.
```

Comment:
Nit: Should we also add that this mode is removed from druid 31.

Reply:
i considered this, but then also thought it was kind of embarrassing that it took so long to remove it, so not really sure we needed to highlight that. Also I can't imagine that many people are using it, i think the only reason to do it is if you were originally using the contrib extension. The only reason I didn't remove it entirely from the docs was just in case people are searching for what it means in native query json stuff have something to reference.

If you think there is some actionable purpose to specify which version it was removed in i can add it, but it doesn't seem too important to me.